kong/db/migrations/core/004_110_to_120.lua

```diff
@@ -0,0 +1,16 @@
+return {
+  postgres = {
+    up = [[
+      DO $$
+      BEGIN
+        CREATE INDEX IF NOT EXISTS cluster_events_expire_at_idx ON cluster_events(expire_at);
+      EXCEPTION WHEN UNDEFINED_COLUMN THEN
+        -- Do nothing, accept existing state
+      END$$;
+    ]],
+  },
+
+  cassandra = {
+    up = [[]],
```

Comment:
is this empty entry necessary? can't it be left out all together?

Reply:
Migrations have their own schema, which requires this field: https://github.com/Kong/kong/blob/master/kong/db/schema/others/migrations.lua